frontend/src/lib/components/TaxonomicFilter/InfiniteList.tsx

```diff
@@ -271,6 +275,13 @@ export function InfiniteList(): JSX.Element {
     } = useValues(infiniteListLogic)
     const { onRowsRendered, setIndex, expand, updateRemoteItem } = useActions(infiniteListLogic)
 
+    // if the popper is not enabled then don't start with a row selected
+    useEffect(() => {
+        if (!popperEnabled) {
+            setIndex(NO_ITEM_SELECTED)
+        }
+    }, [popperEnabled])
+
```

Comment:
Hmm... If we're already changing data based on props, we should handle it in the data layer. Can this go in the logic directly?